starters/nextjs/basic/apphosting.yaml

```diff
@@ -0,0 +1,23 @@
+# Settings for Backend (on Cloud Run).
+# See https://firebase.google.com/docs/app-hosting/configure#cloud-run.
+runConfig:
+  minInstances: 0
```

Comment:
why does Angular comment out all values but Nex.tjs does not?

Reply:
I see that you use the variable in Next.js, but then why not use it in Angular SSR?

Reply:
I have no idea how to use env vars in angular. If you do, welcoming a PR!

Reply:
process.env doesn't work?

Reply:
no